contracts/bridge/BridgeOperator.sol

```diff
@@ -68,11 +71,35 @@ contract BridgeOperator is Ownable {
     {
         require(configurationNonce == _requestNonce, "nonce mismatch");
 
-        if (voteCommon(VoteType.Configuration, _voteKey, _requestNonce)) {
+        if (voteCommon(VoteType.Configuration, _voteKey)) {
             configurationNonce++;
```

Comment:
This line also needs to check overflow.

Reply:
`configurationNonce` is uint64. So I think it doesn't need. :)